apps/exports/views.py

```diff
@@ -178,7 +179,7 @@ def databases(request):
     })
 
 
-@login_required
+@staff_required
```

Comment:
I think you can just replace this with:

`@user_passes_test(lambda u: u.is_superuser)`